files/assets/css/main.css

```diff
@@ -553,9 +553,8 @@ textarea.form-control {
 	user-select: none;
 	background-color: transparent;
 	border: 1px solid transparent;
-	padding: 0.375rem 0.75rem;
```

Comment:
This winds up breaking `.btn` in other contexts, notably the frontpage. 

![image](https://user-images.githubusercontent.com/104547575/189551082-d882524e-0d8e-4f4a-83b1-fe86fffdd83e.png)

It's a fairly over-used class, so perhaps best to leave this class the same and split off a (mostly-duplicated) class for the places you want to change padding & line-height.

Reply:
I actually can't even figure out why I did that; probably a design tweak that became irrelevant after further tweaks. Reverted, and that actually didn't solve it entirely, so, further changes.

this css is a mess but I don't even know what non-messy CSS looks like.